sql1.txt

```diff
@@ -5,25 +5,37 @@ SELECT * FROM Products LIMIT 5;
 
 1. Use an SQL SELECT statement to retrieve the first 10 rows of the Customer table, ordered by CustomerName.
 Paste your SQL statement below:
-
+SELECT * FROM customers
+ORDER by CustomerName ASC
+LIMIT 10; 
 
 2. Use an SQL Select statement to retrieve the names and prices for all products that cost less than $20.
 Paste your SQL statement below:
-
+SELECT ProductName, Price 
+FROM Products
+WHERE Price < 20;
 
 3. Retrieve all employees whose last name starts with C.  Paste your SQL statement below:
-
+SELECT FirstName, LastName
+FROM Employees
+WHERE LastName LIKE "C%";
 
 4. Retrieve the order ID and customer name for all orders where the customer name starts with A.
 Here you will have to do a join of the Orders table and the Customers table.  Paste your SQL
 statement below.
-
+SELECT OrderID, CustomerName 
+FROM Customers 
+JOIN Orders
+ON OrderID = Orders.OrderID
```

Comment:
I don't see any change to this answer, but since the same mistake was corrected in the answer 5 below, I can tell you that what you want is:

```suggestion
ON Orders.CustomerID = Customers.CustomerID
```

because `CustomerID` is the ID column that the Customers and Orders tables have in common. `OrderID` exists only on the Orders table, so joining on that is effectively a [cross join](https://en.wikipedia.org/wiki/Join_(SQL)#Cross_join), or a join where the condition is always true. The result becomes each individual order joined with every customer in existence, as opposed to the correct result which is each individual order joined with the individual customer who made that order.